packages/FormElement/stories/examples/ButtonGroup.js

```diff
@@ -1,27 +1,38 @@
 import React from "react";
 import ButtonGroup from "@paprika/button-group";
-import stylers from "@paprika/stylers";
+import L10n from "@paprika/l10n";
 import FormElement from "../../src";
 
-const { Content, Label } = FormElement;
+const { Content, Label, Description } = FormElement;
 
 export default function ButtonGroupExample() {
   const refButtonGroup = React.useRef(null);
-  const buttonOptionsArray = ["Black Panther", "Wonder Woman", "Spiderman"];
-  const buttonGroupOptions = buttonOptionsArray.map(hero => (
-    <ButtonGroup.Item value={hero} key={hero}>
-      {hero}
-    </ButtonGroup.Item>
-  ));
+  const labelId = "label-buttongroup";
+  const buttonOptions = ["Batman", "Wolverine", "Daredevil"];
+
+  function renderGroupOptions(ariaDescribedBy) {
+    return buttonOptions.map(item => (
+      <ButtonGroup.Item value={item} key={item} aria-describedby={ariaDescribedBy}>
+        {item}
+      </ButtonGroup.Item>
+    ));
+  }
 
   return (
-    <FormElement>
-      <Label onClick={() => refButtonGroup.current.focus()}>Form Label</Label>
-      <Content>
-        <ButtonGroup style={{ marginTop: stylers.spacer(2) }} ref={refButtonGroup}>
-          {buttonGroupOptions}
-        </ButtonGroup>
-      </Content>
-    </FormElement>
+    <L10n>
+      <FormElement>
+        <Label onClick={() => refButtonGroup.current.focus()} id={labelId}>
+          Form Label
+        </Label>
+        <Content>
+          {a11yProps => (
+            <ButtonGroup ref={refButtonGroup}>
+              {renderGroupOptions(`${labelId} ${a11yProps["aria-describedby"]}`)}
```

Comment:
how come you don't use the `id` coming from `a11yProps`?

Reply:
The `id` from `a11yProps` is destined for the input element (which we don't have in this case).  This is a workaround for that – I can append the content of the `<label>` to the ARIA description this way (without overriding the accessible name, which should remain the `children` content of the `<ButtonGroup.Item>`.